app/js/renderer.js

```diff
@@ -301,13 +301,21 @@ MathJax.Hub.Register.StartupHook("AsciiMath Jax Ready", function () {
       'plantuml-server-url=http://www.plantuml.com/plantuml@']
     const href = window.location.href
     const fileName = href.split('/').pop()
-    let fileExtension = fileName.split('.').pop()
-    if (fileExtension !== '') {
+    attributes.push(`docfile=${fileName}`)
```

Comment:
`docfile` is the *full path* of the source document. So we should probably use `href` ?

What do you think ?

//ping @mojavelinux

Reply:
changed it to href.